addons/hr_org_chart/models/models.py

```diff
@@ -0,0 +1,44 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
+
+    @api.model
+    def hierarchy_read(self, domain, fields, parent_field, child_field=None):
```

Comment:
All this should be moved to a ``web_org_chart`` module, but considering the time we got left, this could be done afterwards.

Reply:
done 🙂

Reply:
move in `web_hierarchy`